vocode/streaming/agent/utils.py

```diff
@@ -49,13 +49,19 @@ async def collate_response_async(
             possible_list_item = bool(re.match(r"^\d+[ .]", buffer))
             ends_with_money = bool(re.findall(r"\$\d+.$", buffer))
             if re.findall(
-                list_item_ending_pattern
-                if possible_list_item
-                else sentence_endings_pattern,
+                (
+                    list_item_ending_pattern
+                    if possible_list_item
+                    else sentence_endings_pattern
+                ),
                 token,
             ):
                 # Check if the last word in the buffer is longer than 3 letters
                 if not ends_with_money and len(buffer.strip().split()[-1]) >= 4:
+                    # also check that the buffer is longer than 2 words
+                    # prevents clicking from when the audio plays faster than the next chunk returns
```

Comment:
add in because the synthesizer either has a gap in the playback or closes altogether because the chunk is played too quickly